src/tests/system/tests/test_offline.py

```diff
@@ -0,0 +1,96 @@
+"""
+Automation of offline tests
+
+:requirement: offline
+"""
+
+from __future__ import annotations
+
+import time
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.ldap import LDAP
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.ticket(bz=1416150)
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_offline__log_to_syslog(client: Client):
+    """
+    :title: Log to syslog when sssd cannot contact servers goes offline
+    :setup:
+        1. Set sssd.conf properly
+        2. Start SSSD
+    :steps:
+        1. Check domain status for default domain
+        2. Save date and restart SSSD
+        3. Check journalctl
+    :expectedresults:
+        1. Domain is offline
+        2. Succeed
+        3. "Backend is offline" found
+    :customerscenario: True
+    """
+    client.sssd.domain["ldap_uri"] = f"ldaps://typo.{client.host.hostname}"
+    client.sssd.domain["ldap_sudo_random_offset"] = "0"
+    client.sssd.start()
+    time.sleep(5)
```

Comment:
I guess this sleep is similar to the one we have been analyzing at https://github.com/SSSD/sssd/pull/6928. Right?

Reply:
Yes

Reply:
Yes

Reply:
We don't need this sleep anymore, do we?

Reply:
You are right